src/shared/components/header/header.tsx

```diff
@@ -1,7 +1,79 @@
-import React from 'react';
+import i18nConfig from '@/../i18n.json';
 import { SC } from '@/shared/components/header/header.styles';
+import useTranslation from 'next-translate/useTranslation';
+import { useRouter } from 'next/router';
+import { useState } from 'react';
+
+const menuItems = [
+  { id: 1, href: '/', text: 'home' },
+  { id: 2, href: '/donations', text: 'donations' },
+  { id: 3, href: '/vacancies', text: 'vacancies' },
+];
 
 function Header() {
-  return <SC.Container>Header</SC.Container>;
+  const { t, lang } = useTranslation('header');
+
+  const router = useRouter();
+  const [isLanguageMenuOpen, setIsLanguageMenuOpen] = useState(false);
+
+  const toggleLanguageMenu = () => setIsLanguageMenuOpen((prev) => !prev);
+
+  const changeLanguage = (selectedLang: string) => {
+    if (selectedLang !== lang) {
+      setIsLanguageMenuOpen(false);
+      router.push(router.pathname, router.asPath, { locale: selectedLang });
+    }
+  };
+
+  return (
+    <SC.Header>
+      <SC.Container>
+        <SC.TitleLogo>
+          <SC.Logo src="/img/logo.png" />
+          <SC.Title>{t('name')}</SC.Title>
+        </SC.TitleLogo>
+        <SC.Navigation>
+          {menuItems.map((item) => (
+            <SC.NavigationListItem href={item.href} key={item.id}>
```

Comment:
item.id is a siple number that can be duplicated on screen if we use it. 
Change to more unique id.
```suggestion
<SC.NavigationListItem href={item.href} key={`nav-item-${item.tranKey}`}>
```

Reply:
[fix: logic](https://github.com/p-osypov/army/pull/1/commits/d3a4799e2c45e5227a2e90e8afa181fdef37ae04)